go.mod

```diff
@@ -2,6 +2,10 @@ module sigs.k8s.io/cloud-provider-azure
 
 go 1.23.1
 
+godebug winreadlinkvolume=0
```

Comment:
note that we _may_ just be setting the `winsymlink` option in 1.32 in Kubernetes, see discussion in https://github.com/kubernetes/kubernetes/pull/129103

Reply:
thanks, fixed by https://github.com/kubernetes-sigs/cloud-provider-azure/pull/7771